utilities/transactions/write_committed_transaction_ts_test.cc

```diff
@@ -498,6 +498,60 @@ TEST_P(WriteCommittedTxnWithTsTest, RefineReadTimestamp) {
   txn0.reset();
 }
 
+TEST_P(WriteCommittedTxnWithTsTest, CheckKeysForConflicts) {
```

Comment:
Could you add some comments to this test case to make the scenario clearer?

Reply:
Will add.